jiant/tasks/tasks.py

```diff
@@ -1545,15 +1548,25 @@ class RTESuperGLUETask(RTETask):
     def load_data(self):
         """ Process the datasets located at path. """
         targ_map = {"not_entailment": 0, True: 1, False: 0, "entailment": 1}
-        self.train_data_text = load_pair_nli_jsonl(
-            os.path.join(self.path, "train.jsonl"), self._tokenizer_name, self.max_seq_len, targ_map
-        )
```

Comment:
could you change load_pair_nli_json instead of re-inserting it back here? For example, instead of return [sent1s, sent2s, trgs, idxs, pair_ids], perhaps have another condition that only returns the first four items for RTE.

Reply:
I'd rather not rely on a helper function in another file here. It's pretty straightforward to load a JSONL, and it only gets more complicated with all these different task-specific quirks in a single function.

Reply:
Alright - the thinking was basically to reduce the amount of repeated code in the codebase, but up to you!